script/entrypoint.sh

```diff
@@ -2,23 +2,9 @@
 set -ex
 
 # Render a AWS credentials with user and role for `awsudo`.
-DEFAULT_REGION=us-east-1
+DEFAULT_REGION=ap-southeast-1
 REGION=${REGION:-${DEFAULT_REGION}}
```

Comment:
Also add capability to take different profiles, as of now, it will only work for the default aws profile right ?

Reply:
added